src/core/main.lua

```diff
@@ -11,7 +11,6 @@ local ffi = require("ffi")
 local zone = require("jit.zone")
 local lib = require("core.lib")
 local shm = require("core.shm")
-local app = require("core.app")
```

Comment:
Update: when trying to report this upstream I realized it wasn't occuring, on further examination I think I added this import and I can't find it in the `lwaftr` branch or upstream master. Feel a little bit silly now!

Anyway. I think it's fine to remove it as I think I must have added it, causing the bug in the first place.